docs/references/nextjs/route-protection-requiring-authorization.mdx

```diff
@@ -91,60 +82,32 @@ if (!auth().has({ role: "org:admin" })) {
 
 You can use protect queries and mutations in your Route Handlers or Server Actions in two ways:
 
-- `auth().protect()` if you want Clerk to automatically return an error
+- `auth().protect()` if you want Clerk to redirect user who isn't signed in to the sign-in route, or return a 404 if the user does not have the role or permission
 - `auth().has()` if you want to customize the behavior or error message
 
 ```tsx filename="/app/api/dashboard/route.ts"
-import { auth } from "@clerk/nextjs/server";
+import { auth } from '@clerk/nextjs/server';
 
 export async function GET() {
+  if (!auth().has({ permission: 'org:sys_memberships:manage' })) {
+    return Response.json(
+      { error: 'Error: User does not have required permissions' },
+      { status: 401 },
+    );
+  }
 
-  // If there is no signed in user, this will return an error
-  auth().protect({ permission: "org:sys_memberships:manage" })
-
-  // Perform your Route Handler logic here
+  // Add your Route Handler logic here
 
-  return Response.json({ userId: auth().userId })
+  return Response.json({ userId: auth().userId });
 }
 ```
 
 ## Pages Router
 
 ### Protect a page
 
-You can protect a page using [`getServerSideProps()`](https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props) when using Server Side Rendering, or with [`useAuth()`](/docs/references/react/use-auth) from the client side.
-
-<Tabs type="router" items={["Server Side Rendered", "Client side"]}>
-  <Tab>
-  {/* TODO: Add getServerSideProps example */}
-  </Tab>
+`getAuth()` does not provide access to the `has()` method. We recommend that you handle authorization for routes in the Middleware, where `clerkMiddleware()` will provide access to `auth().protect()` and `auth().has()`.
```

Comment:
wait, I thought has is available on the Auth object - so why doesn't getAuth provide access to has?

and the example before was using useAuth - so why did we remove it?